config/routes.rb

```diff
@@ -3,9 +3,17 @@
   devise_for :users
 
   resources :stations, only: [:index, :show, :new, :create] do
-    resources :reviews, except: [:index, :destroy]
+    resources :reviews, except: [:index, :destroy] do
+    end
   end
 
+  resources :reviews, only: [] do
+    resources :votes, only: [:destroy]
+    get '/up-vote', to: 'votes#up_vote', as: :up_vote
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.